ai-service/src/ai_service/project_ingestor.py

```
@@ -0,0 +1,60 @@
+import os
+import tempfile
+import shutil
+from typing import List, Dict
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)